src/main/java/org/kohsuke/github/GHObject.java

```diff
@@ -26,10 +29,6 @@ public Date getCreatedAt() throws IOException {
         return GitHub.parseDate(created_at);
     }
 
-    private Object createdAtStr(Date id, Class type) {
-        return created_at;
-    }
```

Comment:
Yes. I think it must not be removed (or remove both the method and the annotation).